### PR TITLE
Speech to text transcription model using deep speech

### LINKED ISSUE #6088: [Machine Learning] Speech to text transcription model using deep speech
#### Proposed title of article
[Machine Learning] Speech to text transcription model using deep speech

#### Proposed article introduction
Speech-to-text transcription is a subset of natural language processing that is used to convert speech into text. Speech may be in form of video files and audio files,  the model analyses the speech and converts it to the corresponding text. Speech to text is used to recognize and translate spoken language into text by making use of computational linguistics and understanding.

It is used in various areas such as customer service, medical speech diagnosis to convert spoken words to text. This helps improve customer feedback and suggestion. This model makes consumers more comfortable relying on voice services and businesses can now integrate speech recognition to promote growth and lower operating costs.

In this tutorial, we will use the deep speech library to perform speech-to-text transcription. DeepSpeech takes digital audio as input and returns a text transcript of that audio.

#### Key takeaways 
1. Speech to text transcription?
2. Speech-to-text transcription modern applications.
3. Installing deep speech AI
4. Audio input data pre-processing.
5. Modeling training and text output.

#### Article quality
This tutorial is unique because we will first go through the basics of speech-text transcription. It will also cover some of the applications of speech-to-text transcription, this enables the reader to understand the real word use of this model. Finally, we will use a custom input audio and implement the conversion automatically. The model will output a text that represents the original input audio. 
#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by francis966**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #6088

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

File: content/articles/speech-to-text-transcription-model-using-deep-speech/index.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+This code will produce a text transcript in real-time. The output is shown below:
+
+![Text transcription output](/engineering-education/speech-to-text-transcription-model-using-deep-speech/text-transcription.jpg)
+
+From the output above we can see the model was able to do real-time transcription. The model produced an output bit-by-bit as the audio processing continued. The acoustic and language models produce an accurate transcription.
+
+### Conclusion
+In this tutorial, we have learned how to do a streaming transcription using Deep Speech. We discussed the types of transcription models. We explored the types of pre-trained Deep Speech models. We use the acoustic and language models to build a model that was able to do real-time transcription.
+
+To access this notebook, click [here](https://colab.research.google.com/drive/15amwA8CdN9AApOoZGnYS3roNMxr6mAcy?usp=sharing)
```
Comment (collins-a) [inline content/articles/speech-to-text-transcription-model-using-deep-speech/index.md:264]: Ensure this is the correct Colab notebook. The linked notebook is on anomaly detection and not transcription

**Review by collins-a — CHANGES_REQUESTED** [review]
@francis966 great tutorial. Sort out the Colab notebook then ping me to complete the review.

**Comment by francis966** [comment]
Hello @collins-a, I have provided the right link.
Thanks.

**Review by collins-a — APPROVED** [review]

**Comment by ahmadmardeni1** [comment]
Great work team! @francis966 @collins-a

**Pull request merged**